pkg/controller/operators/olm/operatorgroup.go

```diff
@@ -418,6 +428,10 @@ func (a *Operator) ensureTenantRBAC(operatorNamespace, targetNamespace string, c
 		return err
 	}
 
+	if len(ownedRoles) == 0 {
```

Comment:
What makes this worth checking, but not the targetCSV? Perhaps this PR obsoletes https://github.com/operator-framework/operator-lifecycle-manager/pull/837/, but the above was the biggest cache issue I saw.